app/renderer/components/navigation/navigator.js

```diff
@@ -54,68 +55,24 @@ class Navigator extends ImmutableComponent {
   }
 
   onNav (e, navCheckProp, navType, navAction) {
-    const activeFrame = frameStateUtil.getActiveFrame(this.props.windowState)
-    const activeTab = tabState.getActiveTab(this.props.appState)
-    const activeTabId = tabState.getActiveTabId(this.props.appState)
-    const isNavigable = isNavigatableAboutPage(getBaseUrl(activeFrame.get('location')))
-    if (e && eventUtil.isForSecondaryAction(e) && isNavigable) {
-      if (activeTab && activeTab.get(navCheckProp)) {
-        appActions.tabCloned(activeTabId, {
+    if (e && eventUtil.isForSecondaryAction(e) && this.props.isNavigable) {
+      if (this.props.get(navCheckProp)) {
```

Comment:
shouldn't this be `this.props[navCheckProp]`? Is there a test that covers this because I think this would cause an exception